js/main.js

```diff
@@ -1,2 +1,40 @@
 /* global data */
 /* exported data */
+
+var urlInput = document.querySelector('.photoUrl');
+var imgSrc = document.querySelector('.placeholder-img');
+urlInput.addEventListener('input', function (event) {
+
+  var imgUrl = event.target.value;
+  imgSrc.setAttribute('src', imgUrl);
+
+});
+
+var submitForm = document.getElementById('codeform');
+
+function submitEntry(event) {
+
+  event.preventDefault();
+
+  var formEntry = {
+    title: event.target[0].value,
+    photoUrl: event.target[1].value,
+    notes: event.target[2].value
```

Comment:
You should work off of the `elements` property on the `submitForm` variable

Reply:
Also, you should be adding a property named `entryId` with the value of the `nextEntryId` property on `data` as its value before you increment it.